Makefile

```diff
@@ -69,16 +69,6 @@ clean:
 show-args:
 	@echo "VERSION: $(VERSION)"
 
-# If you setup your laptop following https://github.com/confluentinc/cc-documentation/blob/master/Operations/Laptop%20Setup.md
-# then assuming caas.sh lives here should be fine
-define aws-authenticate
-	source ~/git/go/src/github.com/confluentinc/cc-dotfiles/caas.sh && if ! aws sts get-caller-identity; then eval $$(gimme-aws-creds --output-format export --roles "arn:aws:iam::050879227952:role/administrator"); fi
-endef
-
-.PHONY: fmt
-fmt:
```

Comment:
Every IDE I can think of does this automatically now. I haven't used this in a few years.